api/hackerLive.js

```diff
@@ -157,30 +157,30 @@ router.get("/tasks", async (req, res) => {
   return res.json({ success: tasks });
 });
 
-router.get("/houseInfo/:id", async (req, res) => {
-  const houseId = req.params.id;
```

Comment:
switched the order of `/houseInfo/:id` and `/houseInfo/list` because `/:id` being before `/list` messed things up

Reply:
Oh does it do in order eval?

Reply:
yeah i think so